upstream fix to texcount to allow for verb+foobar+ constructs
(Closes: #711491)
---
 texmf-dist/scripts/latex2man/latex2man |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- texlive-extra-2013.20130509.orig/texmf-dist/scripts/texcount/texcount.pl
+++ texlive-extra-2013.20130509/texmf-dist/scripts/texcount/texcount.pl
@@ -1541,7 +1541,7 @@
  }
  my $dlm=$&;
  print_style($dlm,'command');
- if (!($tex->{'line'} =~ s/^([^$dlm]*)($dlm)// )) {
+ if (!($tex->{'line'} =~ s/^([^$dlm]*)([$dlm])// )) {
   error($tex,'Invalid \verb: could not find ending delimiter ('.$dlm.').');
  }
  print_style($1,'ignore');
